src/OpenLoco/src/Objects/TreeObject.cpp

```diff
@@ -195,10 +195,15 @@ namespace OpenLoco
                 sprites[1][variant] = sprites[0][variant] + numPrimaryImages;
```

Comment:
Should we maybe split the sprites array into snow and normal?

Reply:
I agree that would be more readable, yeah. Suggestions for new names? `normalSprites` and `snowSprites`?

Reply:
could just go `sprites` and `snowSprites`. Have a little check to see how over users of it use it.

Reply:
I imagine it'll only be mostly used by the paint routines, which aren't implemented yet.